src/driver/imu/config/ctrl4c.rs

```diff
@@ -0,0 +1,177 @@
+use super::{Register, RegisterSetting};
+
+pub struct Ctrl4C(
+    pub XlBwScalOdr,
+    pub SleepG,
+    pub Int2OnInt1,
+    pub FifoTempEn,
+    pub DrdyMask,
+    pub I2cDisable,
+    pub StopOnFth,
+);
+
+impl Register for Ctrl4C {
+    fn address(&self) -> u8 {
+        0x13
+    }
+
+    fn value(&self) -> u8 {
+        let Self(xlbwscalodr, sleepg, int2onint1, fifotempen, drdymask, i2cdisable, stoponfth) =
+            self;
+
+        xlbwscalodr.value()
+            | sleepg.value()
+            | int2onint1.value()
+            | fifotempen.value()
+            | drdymask.value()
+            | i2cdisable.value()
+            | stoponfth.value()
+    }
+}
+
+pub enum XlBwScalOdr {
+    /// Bandwidth determined by ODR selection
+    ByOdr,
+
+    /// Bandwidth determined by setting [`BwXl`] in [`Ctrl1Xl`](super::ctrl1xl::Ctrl1Xl)
+    ByBwXl,
+}
+
+impl RegisterSetting for XlBwScalOdr {
+    fn value(&self) -> u8 {
+        match self {
+            Self::ByOdr => 0b_0000_0000,
+            Self::ByBwXl => 0b_1000_0000,
+        }
+    }
+    fn mask(&self) -> u8 {
+        0b_1000_0000
+    }
+}
+
+pub enum SleepG {
+    /// Gyroscope sleep mode enabled
+    GyroEnable,
+
+    /// Gyroscope sleep mode disabled
+    GyroDisable,
+}
```

Comment:
Sorry I missed this before. These names and comments don't match up. "Gyroscope sleep mode enabled" and `GyroEnable` are opposites.

Since this has been confusing in the past, I wonder if we should also change the names from `GyroEnable`/`GyroDisable` to something like `GyroWake`/`GyroSleep`?